OpenRA.Game/Map/MapGrid.cs

```diff
@@ -13,11 +13,11 @@
 
 namespace OpenRA
 {
-	public enum TileShape { Rectangle, Diamond }
+	public enum MapGridType { Rectangular, RectangularIsometric }
```

Comment:
Do we have anything other than these two?
`RectangularIsometric` looks like a change for the sake of making a change.